src/gardener/gardener-tree.js

```diff
@@ -280,6 +295,37 @@ function plantTooltip (element) {
   return list.join('\n')
 }
 
+async function backupinfras ({ parent: project }) {
+  const backupInfra = new BackupInfraClient(project.landscape.kubeconfig, project.namespace)
+  const backupInfras = await backupInfra.list()
+  return _.map(backupInfras, backupInfra => {
+    return toBackupInfraTreeNode(project, backupInfra)
+  })
+}
+
+function toBackupInfraTreeNode (project, backupInfra) {
+  const name = _.get(backupInfra, 'metadata.name')
+  return {
+    nodeType: nodeType.NODE_TYPE_BACKUPINFRA,
+    kindPlural: 'backupinfrastructures',
+    name,
+    project,
+    seed: _.get(backupInfra, 'spec.seed'),
+    forceDeletionEnabled: _.get(backupInfra, ['metadata', 'annotations', 'backupinfrastructure.garden.sapcloud.io/force-deletion']) || "false",
```

Comment:
the lodash get function supports a default parameter
```suggestion
    forceDeletionEnabled: _.get(backupInfra, ['metadata', 'annotations', 'backupinfrastructure.garden.sapcloud.io/force-deletion'], "false"),
```